.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
     env: BUILDTYPE=Release
 script: "./travis-compile.sh"
 install: "./travis-dependencies.sh"
-cache: apt
+cache: ccache
```

Comment:
So, just adding the line `cache: ccache` it seems to affect the travis environment:
Without:
```
$ export CXX=g++
$ export CC=gcc
....
-- The C compiler identification is GNU 4.8.4
-- The CXX compiler identification is GNU 4.8.4
-- Check for working C compiler: /usr/bin/gcc
-- Check for working C compiler: /usr/bin/gcc -- works
```

With `cache: ccache`:
```
$ export CXX=g++
$ export CC=gcc
$ export PATH=/usr/lib/**ccache**:$PATH
cache.1
Setting up build cache
....
-- The C compiler identification is GNU 4.8.4
-- The CXX compiler identification is GNU 4.8.4
-- Check for working C compiler: /usr/lib/**ccache**/gcc
-- Check for working C compiler: /usr/lib/**ccache**/gcc -- works
```

This is actually quite cool!
It doesn't seems to work for OSX builds, anyway.